docs/docsource/installation/upgrade/xdm_migrate_from_kubedb_to_bitnami.rst

```diff
@@ -0,0 +1,282 @@
+- Updating the `MongoDB` replica set configuration.
+
+.. warning::
+
+   The upgrade procedure will result in |product| instance downtime, and the
+   following services will not be operational:
+
+   - Cloudserver S3 APIs
+   - Backbeat APIs
+   - Orbit Management APIs
+   - Vault Management APIs
+
+Update the ``Zenko-Operator``
+-----------------------------
+
+#. Update the ``zenko-operator`` deployment:
+
+   .. code::
+
+      /srv/scality/metalk8s-2.9.2/solution.sh add-solution -n zenko -s zenko -V 2.x.x
+
+   After a few seconds you should see the new operator running. From there you can update each components.
+
+Deploy the ``Bitnami MongoDB``
+------------------------------
+
+#. Create the `secret` resource manifest, ``mongodb-creds.yaml``
+
+   .. code:: 
+
+      MONGODB_ROOT_USERNAME=$(kubectl -n zenko get secret artesca-data-base-db-auth -o jsonpath='{.data.username}')
+      MONGODB_ROOT_PASSWORD=$(kubectl -n zenko get secret artesca-data-base-db-auth -o jsonpath='{.data.password}')
+      MONGODB_USERNAME=$(kubectl -n zenko get secret artesca-data-db-app-creds.v2 -o jsonpath='{.data.username}')
+      MONGODB_PASSWORD=$(kubectl -n zenko get secret artesca-data-db-app-creds.v2 -o jsonpath='{.data.password}')
+      MONGODB_REPLICA_SET_KEY=$(kubectl -n zenko get secret artesca-data-base-db-cert -o jsonpath='{.data.key\.txt}')
+      INSTANCE_ID=$(kubectl --kubeconfig /etc/kubernetes/admin.conf -n zenko get zenko/artesca-data -o jsonpath='{.status.instanceID}' | base64)
+      cat >> mongodb-creds.yaml <<EOF
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: mongodb-db-creds
+      data:
+        mongodb-root-username: $MONGODB_ROOT_USERNAME
+        mongodb-root-password: $MONGODB_ROOT_PASSWORD
+        mongodb-username: $MONGODB_USERNAME
+        mongodb-password: $MONGODB_PASSWORD
+        mongodb-database: $INSTANCE_ID
+        mongodb-replica-set-key: $MONGODB_REPLICA_SET_KEY
+      EOF
+
+   - "mongodb-root-username"  is the superadmin user name, "root".
+   - "mongodb-root-password" is the superadmin user password.
+   - "mongodb-username" is the zenko application user name.
+   - "mongodb-password" is the zenko application user password.
+   - "mongodb-database" is the database the zenko user is allowed to access;
+     this value is the instance id of the zenko instance.
+   - "mongodb-replica-set-key" are the credentials used for allowing members to
+     be added to a replica set.
+
+#. Copy either the `mongodb-1-node.yaml` or `mongodb-3-nodes.yaml` into a new
+   file called `mongodb.yaml`
+
+#. Edit `mongodb.yaml` and replace the namespace placeholder, `SOLUTION_ENV`,
+   with the value, "zenko".
+
+#. Edit `mongodb.yaml` and replace the storageClass placehodler,
+   `MONGODB_STORAGE_CLASS`, with with the existing PV storage class type
+   retrieved in the following manner:
+
+   .. code::
+
+      kubectl get pv -l app.kubernetes.io/name=mongodb,app.kubernetes.io/part-of=zenko 
```

Comment:
This command can be tweaked to output only the storageclass, the output can be intimidating to non-engineers.

Reply:
* updated commands to only list name and storageclass